h2o-core/src/main/java/hex/ModelMetricsBinomialGLM.java

```diff
@@ -109,6 +119,15 @@ public String toString() {
           _nullDegressOfFreedom     == mm._nullDegressOfFreedom     &&
           Math.abs(_resDev - mm._resDev) < 1e-8;
     }
+
+    @Override
+    public ComparisonUtils.AccumulatedComparisonResult isEqualUpToTolerance(ModelMetrics other, double proportionalTolerance) {
```

Comment:
I think proportionalTolerance is more commonly known as relative tolerance

Reply:
ok, I will rename it, thanks!